server/src/test/java/com/linecorp/centraldogma/server/internal/storage/repository/DefaultMetaRepositoryTest.java

```diff
@@ -89,7 +90,7 @@
     @BeforeClass
     public static void init() throws Exception {
         pm = new DefaultProjectManager(rootDir.getRoot(), ForkJoinPool.commonPool(),
-                                       NoopMeterRegistry.get(), null);
+                                       Executors.newSingleThreadExecutor(), NoopMeterRegistry.get(), null);
```

Comment:
How about using `MoreExecutors.directExecutor()` to avoid a dangling thread?

Reply:
fix it :-)